lib/assertions/is-array-like.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+var isArguments = require("lodash.isarguments");
+var actualAndTypeOfMessageValues = require("../actual-and-type-of-message-values");
+
+module.exports = function (referee) {
+    function isArrayLike(object) {
+        return Array.isArray(object) ||
+            (!!object && typeof object.length === "number" &&
+            typeof object.splice === "function") ||
+            isArguments(object);
+    }
+
+    referee.isArrayLike = isArrayLike;
```

Comment:
Is this necessary? This seems unlike the previous plugins. In this case you're writing directly to `referee` as opposed to using the `referee.add` interface. Tests pass when removing this line.

Reply:
I am trying not to break the public API ... yet, so we can still merge in all these changes to `master`, and still make a release.